cfme/tests/infrastructure/test_publish_vm_to_template.py

```diff
@@ -19,8 +19,15 @@ def vm_crud(provider):
     collection = provider.appliance.provider_based_collection(provider)
     vm_name = random_vm_name(context='pblsh')
     vm = collection.instantiate(vm_name, provider)
-    deploy_template(vm.provider.key, vm_name, provider.data['small_template'],
-                    timeout=2500)
+    try:
```

Comment:
I think it be good to put a `required_field` in the `pytestmark` requiring `small_template`. 

Example:
`required_fields=[['templates', 'small_template']]`

Reply:
@kedark3 no need to catch `KeyError` as your required_fields will take care of this. what you say?